File: internal/content/testdata/bench/rust/smallest-multiple-ending-zeros.rs
fn gcd(a: i64, b: i64) -> i64 {
    let mut a = a;
    let mut b = b;
    while b != 0 {
        let t = b;
        b = a % b;
        a = t;
    }
    a
}

fn smallest_trailing_zero_multiple(a: i64, b: i64) -> i64 {
    let mut power: i64 = 1;
    for _ in 0..b {
        power *= 10;
    }
    (a / gcd(a, power)) * power
}

File: internal/content/testdata/bench/go/smallest-multiple-ending-zeros.go
func gcd(a, b int) int {
	for b != 0 {
		a, b = b, a%b
	}
	return a
}

func smallest_trailing_zero_multiple(a int, b int) int {
	power := 1
	for i := 0; i < b; i++ {
		power *= 10
	}
	return (a / gcd(a, power)) * power
}
